### PR TITLE
Add debugger support for Cloud Run. See b/112087600 for context

### PR DESCRIPTION
**Pull request by hrasadi**
(Partially) Fixes b/112087600.

- [✓] Tests and linter pass
- [✓] Code coverage does not decrease (if any source code was changed)
- [✓] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: src/agent/debuglet.ts [hunk 1]
```diff
@@ -274,8 +274,10 @@ export class Debuglet extends EventEmitter {
     const envConfig = {
       logLevel: process.env.GCLOUD_DEBUG_LOGLEVEL,
       serviceContext: {
-        service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME,
-        version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION,
+        service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME ||
```
Comment (DominicKramer) [inline src/agent/debuglet.ts]: Should `K_SERVICE` be of lowest priority or first here?  Is it possible for `GAE_MODULE_NAME` and `K_SERVICE` to both be defined but have different values?

Comment (hrasadi) [reply src/agent/debuglet.ts]: By definition, GAE and Run -specific keys should not be set at the same time. These are two very separate products so the order should not really matter. I will however track this down with Cloud Run team to make sure this is the case.

Comment (DominicKramer) [reply src/agent/debuglet.ts]: Sounds good.  Thanks.

Comment (DominicKramer) [reply src/agent/debuglet.ts]: @hrasadi have you talked with the Cloud Run team to determine if this can land?

Comment (hrasadi) [reply src/agent/debuglet.ts]: This can land. GAE and Cloud Run env vars will not clash.

Comment (DominicKramer) [reply src/agent/debuglet.ts]: Great.  Thanks for looking into this.

**Pull request merged**